zingo-memo/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub fn create_wallet_internal_memo_version_0(uas: &[UnifiedAddress]) -> io::Resu
     let mut uas_bytes_vec = Vec::new();
     CompactSize::write(&mut uas_bytes_vec, 0usize)?;
     Vector::write(&mut uas_bytes_vec, uas, |w, ua| {
-        write_unified_address_to_raw_encoding(ua, w)
+        write_unified_address_to_raw_encoding(ua, w.clone())
```

Comment:
Does this change have something to do with the Rust drop?

Reply:
yes

Reply:
all this PR does is follow compiler instructions